dynamic-layers/openembedded-layer/recipes-devtools/openocd/openocd_riscv.bb

```diff
@@ -9,6 +9,4 @@ SRC_URI = " \
     git://repo.or.cz/r/libjaylink.git;protocol=http;destsuffix=git/src/jtag/drivers/libjaylink;name=libjaylink \
 "
 
-SRCREV_openocd = "f93ede5401c711e55d9852986aa399c0318efb22"
-
-COMPATIBLE_HOST = "(riscv32|riscv64).*-linux"
+SRCREV_openocd = "3322dfa5e7aa56f5b12b7f2197e3e6b850b7b66e"
```

Comment:
Please leave the COMPATIBLE_HOST in there, and add the additional platforms you need. This helps meta-riscv in good shape to mix with other BSP layers and world builds.